Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -647,6 +649,13 @@ ppExplanation CVAutogenPaths =
       ++ "the module does not come with the package and is generated on "
       ++ "setup. Modules built with a custom Setup.hs script also go here "
       ++ "to ensure that commands like sdist don't fail."
+ppExplanation CVAutogenPackageInfo =
+    "Packages using 'cabal-version: 2.0' and the autogenerated "
+      ++ "module PackageInfo_* must include it also on the 'autogen-modules' field "
+      ++ "besides 'exposed-modules' and 'other-modules'. This specifies that "
```

Comment:
```suggestion
      ++ "module PackageInfo_* must include it in 'autogen-modules' as well as"
      ++ " 'exposed-modules' and 'other-modules'. This specifies that "
```

Reply:
I fixed it, but it's a copy-and-paste from `Paths_*`

Reply:
haha :) The original is what I would consider "European English": used widely among non-native speakers, understandable, but different from what a native speaker would say. I almost considered just leaving it, because I think it's better to say it's dialectical rather than "wrong". Fine either way